src/overlays/actors/ovl_En_Daiku2/z_en_daiku2.h

```diff
@@ -5,9 +5,40 @@
 
 struct EnDaiku2;
 
+typedef void (*EnDaiku2ActionFunc)(struct EnDaiku2*, GlobalContext*);
+
+typedef struct EnDaiku2Particle {
+    /* 0x00 */ u8 enabled;
+    /* 0x04 */ Vec3f unk_04;
+    /* 0x10 */ Vec3f unk_10;
+    /* 0x1C */ Vec3f unk_1C;
+    /* 0x28 */ Vec3s unk_28;
+    /* 0x30 */ f32 unk_30;
+    /* 0x34 */ s16 unk_34;
+} EnDaiku2Particle; // size = 0x38
+
 typedef struct EnDaiku2 {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0xC84];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s limbDrawTable[17];
+    /* 0x1EE */ Vec3s transitionDrawTable[17];
```

Comment:
```suggestion
    /* 0x188 */ Vec3s jointTable[17];
    /* 0x1EE */ Vec3s morphTable[17];
```
More of an open question. These were the names OoT adopted after documentation. MM currently has both the old and new names used throughout the repo, should we start pushing to use the new names?

Reply:
I say yes, and when I get round to doing a significant header-touching PR, I'll probably change all of them anyway. They certainly need to end up consistent, either way.